src/com/cdd/bao/template/Vocabulary.java

```diff
@@ -345,8 +352,17 @@ private void loadLabels(File baseDir, File[] extra, Set<String> exclude) throws
 		files.sort((f1, f2) -> (int)(f1.length() - f2.length()));
 		for (File f : files)
 		{
-			try {RDFDataMgr.read(model, f.getPath(), f.getName().endsWith(".ttl") ? Lang.TURTLE : Lang.RDFXML);}
-			catch (Exception ex) {throw new IOException("Failed to load " + f, ex);}
+			try 
+			{
+				if(f.getName().endsWith(".ttl")) //handles only .ttl files (Jason)
```

Comment:
This section doesn't follow logically - needs to be adapted (e.g. removing the conditional in the first branch). Also we suspected that the problem could be fixed by turning the file path into a file URL, which would be more elegant. Does this not work?

Reply:
figuring out how to pull back down your changes and I'll try to implement the option that Peter suggested.